Taskfile.yml

```diff
@@ -16,7 +16,8 @@ vars:
   RACE_FLAG: -race={{and (ne OS "windows") (ne ARCH "arm") (ne ARCH "riscv64")}}
   BUILD_TAGS: ferretdb_dev
   MODE: diff-normal
-  SERVICES: jaeger mongodb mongodb-secure postgres yugabytedb
+  # TODO https://github.com/FerretDB/FerretDB/issues/5369
+  SERVICES: jaeger mongodb mongodb-secure postgres {{if (ne ARCH "arm64")}}yugabytedb{{end}}
```

Comment:
I don't think we need that complication. It will try to start and fail, but that by itself does not break anything

Reply:
Sure, let's remove this complication